src/main/java/frc/robot/Ports.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot;
+
+public class Ports {
+
+    public static class Climber {
+        public static final int MASTER = 0;
+
+        public static final int SLAVE = 0;
+
+        public static final int STOPPER = 0;
+
+        public static final int GEARBOX_PISTON = 0;
+
+        public static final boolean MASTER_INVERTED = false;

```

Comment:
```suggestion
        public static final boolean IS_MASTER_INVERTED = false;
```
Booleans may start with "is", "has", "does", etc.
The sub-team should agree on a convention (@orel-david).